jwt/jwt_test.go

```diff
@@ -87,3 +88,26 @@ func TestVerify_BadClaims(t *testing.T) {
 	assert.Error(t, err)
 	assert.False(t, verified, "expected !verified")
 }
+
+func TestDecodeJWTClaims(t *testing.T) {
+	did, err := didjwk.Create()
+	assert.NoError(t, err)
+
+	nonce := "abcd123"
+	claims := jwt.Claims{
+		Issuer: did.ID,
+		Misc:   map[string]interface{}{"c_nonce": nonce},
+	}
+
+	signedJwt, err := jwt.Sign(claims, did)
+	assert.NoError(t, err)
+
+	parts := strings.Split(signedJwt, ".")
+	assert.Equal(t, len(parts), 3, "expected 3 parts in JWT")
```

Comment:
Not a big deal but for future reference, the parameters to Equal and NotEqual are `expected, actual`, so if you put the _expected_ value first, the output diff will look correct. It will still work fine this way though.

Reply:
Nice, we have this wrong in other places. Fixed here, opened a ticket for the others https://github.com/TBD54566975/web5-go/issues/27